docker/seahorn.Dockerfile

```diff
@@ -0,0 +1,42 @@
+#
+# Minimal Dockerfile for SeaHorn
+# produces a lightweight container with SeaHorn
+# Arguments:
+#  - UBUNTU:     trusty, xenial
+#
+
+ARG UBUNTU
+FROM buildpack-deps:$UBUNTU
```

Comment:
should depend on a seahorn container that already includes the minimal dependencies, such as clang.
Ideally, this container has everything that is needed to just unpack seahorn binary and go